### PR TITLE
Fix duplicated "the" in pipline aggregation docs

### PR DESCRIPTION
**Pull request by lukaromih**
### Description
Fix minor typo
 
### Issues Resolved
none
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by lukaromih** [comment]
My previous PR [(446)](https://github.com/opensearch-project/documentation-website/pull/466) didn't have a sign-off, so I amended the commit with the -s flag. For that I had to clone the code locally. It took more time and effort.

This time I tried to bypass that step and was hoping to be able to make such a minor fix right from the GitHub's web interface/code editor by copy pasting my (successfully approved) sign-off from the former PR right into the details field of the commit. Sadly the DCO bot doesn't seem to approve of it and I can't find the difference between the sign-off statements of this and the former PR.

**Comment by keithhc2** [comment]
@lukaromih Are you using the web editor to make commits? That method can be a little iffy, but there is a [Chrome extension](https://github.com/scottrigby/dco-gh-ui) that you can use. If you prefer using the CLI, `git commit -s` should do the trick whenever you commit. The DCO check runs off of the entire branch, so if there's even one commit in your branch that doesn't have the sign-off, the whole thing fails. 

Also, for housekeeping purposes, perhaps you can continue committing to the same PR as you find and fix typos? We can prevent things from becoming too unruly that way.

**Comment by lukaromih** [comment]
I fixed the DCO issue (and may have found the reason for failure, but not yet sure).

@keithhc2 Thanks for the heads up about the extension. Will keep it in mind if the workflow requirements call for it in the future.
Agreed about housekeeping. I even wanted to cherry pick this commit on top of the one from PR 446 and close this one. But then:

1. I'd have to rename it into something more generic
2. How would you or I know, when enough typos (or all that I mean to fix in a short time period) have been gathered and the PR can be finally be closed and merged?

**Comment by keithhc2** [comment]
@lukaromih I don't think typos are something we'll ever be rid of completely 😄 so it's really just a judgment call at that point. You could create your PRs as drafts and then switch them over at, say, once a week, or when you feel like they're at a more ready state, and we can merge them on a weekly basis.

**Review by Naarcha-AWS — APPROVED** [review]

**Comment by lukaromih** [comment]
@keithhc2 I've added, what I think is the last commit in this "batch" of typo fixes. I'm not done with documentation yet, though I'll be using it more actively in a few weeks again. I'll make sure to add anything I might find onto my personal list, then if/when things add up, I'll make sure to follow your input from this PR (draft PRs, etc.).

So unless I need to tweak anything or make a rebase, you can use and close this PR.
Also this PR includes the commit included in [PR-466](https://github.com/opensearch-project/documentation-website/pull/466) + 2 others on top of it, so you might as well close that one as well.
As per your guidance, I will not create separate PRs in the future for such trivial fixes.

File: _opensearch/index-templates.md [hunk 1]
```diff
@@ -330,15 +324,14 @@ GET logs-2020-01-01
...
-`version` | `Integer` | Specify a version number to simplify template management. Default is `null`. | No
-`_meta ` | `Object` | Specify meta information about the template. | No
+| Option        | Type           | Description                                                                                 | Required |
+| :------------ | :------------- | :------------------------------------------------------------------------------------------ | :------- |
+| `template`    | `Object`       | Specify index settings, mappings, and aliases.                                              | No       |
+| `priority`    | `Integer`      | The priority of the index template.                                                         | No       |
+| `composed_of` | `String array` | The names of component templates applied on a new index together with the current template. | No       |
+| `version`     | `Integer`      | Specify a version number to simplify template management. Default is `null`.                | No       |
+| `_meta `      | `Object`       | Specify meta information about the template.                                                | No       |
```
Comment (keithhc2) [inline _opensearch/index-templates.md:337]: It looks like the new table works, but all the spaces can make editing the file a bit cumbersome, especially when opened in a code editor. Would you mind changing it back to how it was before?

**Review by keithhc2 — CHANGES_REQUESTED** [review]

File: _opensearch/index-templates.md (hunk 1 above)
Comment (lukaromih) [reply _opensearch/index-templates.md]: I'm sorry.
Those lines were unintentionally changed by the code formatter and I'm somewhat embarrassed they slipped into the commit.
I've replaced the commit with one without the unwanted changes.

**Review by keithhc2 — APPROVED** [review]

File: _opensearch/index-templates.md (hunk 1 above)
Comment (keithhc2) [reply _opensearch/index-templates.md]: No worries, thanks for updating it!

**Pull request merged**